modules/openapi-generator/src/main/resources/rust/model.mustache

```diff
@@ -41,9 +41,9 @@ impl Default for {{{classname}}} {
 {{!-- for schemas that have a discriminator --}}
 {{#discriminator}}
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
-#[serde(tag = "{{{vendorExtensions.x-tag-name}}}")]
+#[serde(tag = "{{{model.vendorExtensions.x-tag-name}}}")]
```

Comment:
Looks like this has been producing `#[serde(tag = "")]` since #15975. The new test should prevent future regressions here.